README.md

```diff
@@ -596,6 +598,33 @@ The DLP templates used in this tutorial are specifically tailored for inspecting
 2. [Create de-identification templates and run de-identification on sample data](https://cloud.google.com/dlp/docs/creating-templates-deid)
 
 
+## Run this pipeline outside Google Cloud shell
+
+1. Create a new GCP project.
+2. [Set up Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc).
+3. Set the project in your active configuration:
+```commandline
```

Comment:
Are there any pre-requisites?
gcloud cli needs to be installed,
java
gradle

We can link to another section of documentation if it is already captured somewhere.

Reply:
Installation of gcloud cli is covered in "Set up ADC" step. Will add other pre-requisites.